src/meta/src/manager/catalog.rs

```diff
@@ -511,7 +514,7 @@ where
 
             for table in tables {
                 core.add_table(&table);
-                self.broadcast_info_op(Operation::Add, Info::Table(table.to_owned()))
+                self.notify_frontend_info_op(Operation::Add, Info::Table(table.to_owned()))
```

Comment:
How about calling `notify_frontend` directly here?

Reply:
ok, i wil rename it and unify the `notify_frontend`